src/util/tls.ts

```diff
@@ -174,7 +174,7 @@ export class CA {
         }
     }
 
-    generateCertificate(domain: string): GeneratedCertificate {
+    generateCertificate(domain: string, countryName?: string, localityName?: string, organizationName?: string): GeneratedCertificate {
```

Comment:
Can we include this in the initial parameters for the CA, instead of passing it into every single `generateCertificate` call? It looks like the values will always be the same, so I think that would be a little cleaner.

That means:

* Moving the options from `MockttpHttpsOptions` to `CAOptions`
* Making the CA class take `CAOptions` as a constructor parameter (instead of the 3 fields)
* Handling that correctly in `getCA`

Does that make sense? I think that would mean there'd be no changes required in http-combo-server at all - we just pass the options to getLocal, and they're handled in CA creation, and applied to all certs automatically.

Reply:
Perfect, this all makes sense.  I will work on this tonight.